app/src/main/java/com/anytypeio/anytype/ui/settings/RemoteFilesManageFragment.kt

```diff
@@ -21,10 +21,10 @@ import com.anytypeio.anytype.presentation.widgets.collection.Subscription
 import com.anytypeio.anytype.presentation.widgets.collection.SubscriptionMapper
 import com.anytypeio.anytype.ui.base.navigation
 import com.anytypeio.anytype.ui.dashboard.DeleteAlertFragment
-import com.anytypeio.anytype.ui.settings.remote.RemoteStorageScreen
+import com.anytypeio.anytype.ui.settings.remote.RemoteFilesManageScreen
 import javax.inject.Inject
 
-class RemoteStorageFragment : BaseBottomSheetComposeFragment() {
+class RemoteFilesManageFragment : BaseBottomSheetComposeFragment() {
```

Comment:
maybe name it `SpaceStorageFragment`?

Reply:
we have already SpacesStorageFragment, can easily be confused